rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -10,9 +10,10 @@
             <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
         </dependencies>
         <sourceTechnology id="hibernate" versionRange="[4,)" />
+        <sourceTechnology id="eap" versionRange="[4,7)" />
```

Comment:
Can we add source technology `eap` also to `hibernate4-xml.windup.xml`?

Also, I don't think this applies to EAP 4 and 5, it should be `<sourceTechnology id="eap" versionRange="[6,7)" />`.